performance_tests/10zone1vav.idf

```diff
@@ -35,10 +35,12 @@
   Output:DebuggingData,0,0;
 
   ZoneCapacitanceMultiplier:ResearchSpecial,
+    1.,                      !- Name
```

Comment:
Just tried to run 20zone1vav.idf and get this:

    ** Severe  ** GetNodeListsInput: NodeList="PVAV_SYS SUPPLY FAN UPSTREAM NODES" has duplicate nodes:
    **   ~~~   ** ...list item=5, "AUTOSIZE", duplicate list item=6, "AUTOSIZE".
    **   ~~~   ** ...list item=5, "AUTOSIZE", duplicate list item=7, "AUTOSIZE".
    **   ~~~   ** ...list item=5, "AUTOSIZE", duplicate list item=8, "AUTOSIZE".

    NodeList,
    PVAV_Sys Supply Fan Upstream Nodes,  !- Name
    COIL:Water:Cooling,      !- Node 1 Name
    PVAV_Sys Cooling Coil,   !- Node 2 Name
    OffFan,                  !- Node 3 Name
    0.022303881,             !- Node 4 Name
    autosize,                !- Node 5 Name
    autosize,                !- Node 6 Name
    autosize,                !- Node 7 Name
    autosize,                !- Node 8 Name
    autosize,                !- Node 9 Name
    autosize,                !- Node 10 Name
    PVAV_Sys Cooling Coil Water Inlet Node,  !- Node 11 Name
    PVAV_Sys Cooling Coil Water Outlet Node,  !- Node 12 Name
    PVAV_Sys Mixed Air Outlet,  !- Node 13 Name
    PVAV_Sys Cooling Coil Outlet,  !- Node 14 Name
    SIMPLEANALYSIS,          !- Node 15 Name
    CROSSFLOW;               !- Node 16 Name